README.md

```diff
@@ -1,30 +1,127 @@
+                    write.write_string(buffer, val_60)
+                
+                elif type_51 == 'int':
+                    write.write_long(buffer, 1)
+                    write.write_int(buffer, val_60)
+                
+
+            else:
+
+                if type(val_59) is str:
+                    write.write_long(buffer, 0)
+                    write.write_string(buffer, val_59)
+
+                elif type(val_59) is int:
+                    write.write_long(buffer, 1)
+                    write.write_int(buffer, val_59)
+
+    write.write_long(buffer, 0)
+    output.write(buffer)
+
+
+
+def deserialize(fo):
+    cdef long long i_62
+    cdef long long i_63
+    cdef long i_64
+    data = {}
+    data['order_id'] = read.read_string(fo)
+    data['trades'] = []
+
+    i_63 = read.read_long(fo)
+    while i_63 != 0:
+        if i_63 < 0:
+            i_63 = -i_63
+            read.read_long(fo)
+        for i_62 in range(i_63):
+            i_64 = read.read_int(fo)
+
+            if i_64 == 0:
+                val_61 = read.read_string(fo)
+
+
+            if i_64 == 1:
+                val_61 = read.read_int(fo)
+
+
+            data['trades'].append(val_61)
+        i_63 = read.read_long(fo)
+    return data
+```
+
+
 **Usage Example:**
-1. Put all your schemata into a schemata root folder following this pattern.<br>
-`[schemata_root_folder]/[namespace]/[schema_name]/[schema_version]/schema.yaml`<br>
+1. Put all your schemata into a schemata root folder following this pattern.
+```
+[schemata_root_folder]/[namespace]/[schema_name]/[schema_version]/schema.yaml
+```
 It is essential that you follow this pattern and safe your schemata in `schema.yaml` files.
 
-2. Run `cerializer_handler.update_cerializer` with a list of all your schemata root directories.<br>
-eg. `cerializer_handler.update_cerializer(['cerializer/tests/schemata'])`<br>
-This will generate an importable module for each schema that contains two methods: `serialize` and `deserialize`.<br>
-This module can be then imported as:<br>
- `import [schema_name]_[schema_version]`<br>
- and then used accordingly<br>
-3. Use the module accordingly.<br>
-  eg.<pre><code>
- import student_schema_1
- import io<br>
+2. Create an instance of Cerializer by calling `cerializer_handler.Cerializer` with a list of all your schemata root directories.
+eg. `cerializer_handler.Cerializer(['cerializer/tests/schemata'])`
+This will create an instance of Cerializer that can serialize and deserialize data in all schema formats.
+
+3. Use the instance accordingly.
+  eg. 
+  ```python
+ import cerializer.cerializer_handler
+ import io
  output = io.BytesIO()
- data_recod = {
+ data_record = {
     'name': 'Matej',
     'age': 22
  }
- student_schema_1.serializer(data_record, output)
+ cerializer_instance = cerializer.cerializer_handler.Cerializer(['schemata'])
+ cerializer_instance.serialize('namespace', 'schema_name', 'schema_version', data_record, output)
```

Comment:
Your `serialize` function now does not have these arguments. This seems outdated

Reply:
My bad... Old readme, sorry for that